packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -58,52 +60,76 @@ class MyHomePage extends StatelessWidget {
             extendBodyBehindAppBar: true,
             body: SizedBox(
               height: MediaQuery.of(context).size.height,
-              child: Container(
-                decoration: BoxDecoration(
-                  gradient: LinearGradient(
-                    begin: Alignment.topCenter,
-                    end: Alignment.bottomCenter,
-                    colors: [
-                      context.pinCodeTheme.primaryGradientStart,
-                      context.pinCodeTheme.primaryGradientEnd
-                    ],
+              child: Column(
+                children: [
+                  Expanded(
+                    child: _buildWithBiometrics(context),
```

Comment:
As discussed the build helper methods should be avoided when possible

Reply:
I have removed the method